bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -0,0 +1,478 @@
+# ovnkube-master
+# daemonset version 3
+# starts master daemons, each in a separate container
+# it is run on the master node(s)
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: ovnkube-master
+  # namespace set up by install
+  namespace: ovn-kubernetes
+  annotations:
+    kubernetes.io/description: |
+      This daemonset launches the ovn-kubernetes networking components.
+spec:
+  selector:
+    matchLabels:
+      name: ovnkube-master
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        name: ovnkube-master
+        component: network
+        type: infra
+        openshift.io/component: network
+        beta.kubernetes.io/os: "linux"
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+    spec:
+      # Requires fairly broad permissions - ability to read all services and network functions as well
+      # as all pods.
+      serviceAccountName: ovn
+      hostNetwork: true
+      hostPID: true
+      #tolerations:
+      #- key: node-role.kubernetes.io/master
+      # effect: Schedule
+      containers:
+      # firewall rules for ovn - assumed to be setup
```

Comment:
Nope. You can't assume that. If you don't set them up, they won't be set up.

Reply:
The operator will need to set up the firewall on the host.

Reply:
If the operator needs to do that, then this PR needs to *make* the operator do that.
One way to do that would be to add an `initContainer` to the pod that runs the necessary iptables commands.

Reply:
I guess for now we can just have the ovnkube init script do this, since it's hostNetwork?